practice-one/pages/index.html

```diff
@@ -0,0 +1,646 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Practice One</title>
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+  <body>
+    <!-- Start Header -->
+    <header class="background-light" id="header">
+      <div class="container">
+        <div class="header-list">
+          <figure>
+            <a href="javascript:void(0)"><img src="../assets/images/logo-digiency.svg" alt="Digiency Logo"></a>
```

Comment:
should have closed tag `<img src="../assets/images/logo-digiency.svg" alt="Digiency Logo"/>`